### PR TITLE
Update Policy Generator YAML table

### PR DESCRIPTION
**Pull request by dhaiducek**
Addresses:
- https://github.com/stolostron/backlog/issues/26898

### CODE REVIEW

**Review by JustinKuli — APPROVED** [review]

**Review by dockerymick — APPROVED** [review]
Made some updates to the file, let me know if you have any questions about my updates. Thanks again for making these changes Dale! Pulling in another writer for a peer review

File: governance/policy_generator.adoc [hunk 1]
```diff
@@ -16,10 +16,7 @@ The policy generator and its integration with the {product-title-short} applicat
 View the following topics to for more information:
 
 * <<policy-generator-configuration,Policy generator configuration structure>>
-//this is a large PR, PTO is coming up, and GA around the corner so commenting my suggestion so that I don't forget
-//To improve architecture of this doc, let's make this easier to read by moving the Generating a policy to install an Operator section + subsections to its own file
-//I also plan to add a reference link to the new file at the end of this file
-//holding off on this change bc links can become broken when creating a new file (file not added to main.adoc, misspelled file name, wrong folder, wrong syntax, etc. 
+//To improve architecture of this doc, let's make this easier to read by moving the Generating a policy to install an Operator section + subsections to its own file. I also plan to add a reference link to the new file at the end of this file | MJ| 11/18
```
Comment (swopebe) [inline governance/policy_generator.adoc]: Hi! Any chance we can remove this comment and put into an issue if one is opened or are you just waiting for the issue at the moment?

Comment (dockerymick) [reply governance/policy_generator.adoc]: I will create the issue. This is my comment

**Review by oafischer — APPROVED** [review]
lgtm

**Comment by dockerymick** [comment]
Thanks @swopebe and @oafischer for the follow up review! Team work makes the dream work :D

**Comment by dockerymick** [comment]
Moving forward with merging this PR

**Pull request merged**